frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/common.unit.spec.tsx

```diff
@@ -1,23 +1,21 @@
+import userEvent from "@testing-library/user-event";
 import {
   createMockSettingDefinition,
   createMockSettings,
 } from "metabase-types/api/mocks";
 import { screen } from "__support__/ui";
 import { setup } from "./setup";
 
-const FULL_APP_EMBEDDING_URL =
-  "/admin/settings/embedding-in-other-applications/full-app";
-
 describe("SettingsEditor", () => {
   describe("full-app embedding", () => {
     it("should show info about full app embedding", async () => {
       setup({
         settings: [createMockSettingDefinition({ key: "enable-embedding" })],
         settingValues: createMockSettings({ "enable-embedding": true }),
-        initialRoute: FULL_APP_EMBEDDING_URL,
       });
 
-      expect(await screen.findByText("Full-app embedding")).toBeInTheDocument();
+      userEvent.click(await screen.findByText("Embedding"));
```

Comment:
Do it as the user would. It exposed several issues in the setup and generally made the test more reliable.